From c7170143fae885fcb89b5ed9d9968bc3e454152e Mon Sep 17 00:00:00 2001
From: Chuck Atkins <chuck.atkins@kitware.com>
Date: Tue, 1 Aug 2017 13:55:44 -0400
Subject: [PATCH] Remove relative paths in cmake pacakge-config

Using relative paths in the cmake config can result in broken
dependencies in downstream packaging when embree is a private
dependency.  This converts the relative paths generated in
embree-config.cmake from /foo/bar/baz/../../libFoo.so to /foo/libFoo.so
---
 common/cmake/embree-config.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/common/cmake/embree-config.cmake b/common/cmake/embree-config.cmake
index 689f6990e..b3d6793a0 100644
--- a/common/cmake/embree-config.cmake
+++ b/common/cmake/embree-config.cmake
@@ -15,6 +15,7 @@
 ## ======================================================================== ##
 
 SET(EMBREE_ROOT_DIR ${CMAKE_CURRENT_LIST_DIR}/@EMBREE_RELATIV_ROOT_DIR@)
+GET_FILENAME_COMPONENT(EMBREE_ROOT_DIR "${EMBREE_ROOT_DIR}" ABSOLUTE)
 
 SET(EMBREE_INCLUDE_DIRS ${EMBREE_ROOT_DIR}/@CMAKE_INSTALL_INCLUDEDIR@)
 SET(EMBREE_LIBRARY ${EMBREE_ROOT_DIR}/@CMAKE_INSTALL_LIBDIR@/@EMBREE_LIBRARY_NAME@)
-- 
2.13.3

